pkg/serializer/jsonstream/json_raw_object_writer.go

```diff
@@ -0,0 +1,101 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+
+package jsonstream
+
+import jsoniter "github.com/json-iterator/go"
+
+// JSONRawObjectWriter contains helper functions to write JSON using the raw API.
+type JSONRawObjectWriter struct {
```

Comment:
I probably wouldn't put this here as this is syntactic sugar wrapper around jsoniter and not a component of the jsonstream serialization. It could be used every time we use jsoniter. I could see it in it's dedicated package in `pkg/util` for ex. WDYT ?

Reply:
It is a good idea.

Reply:
Done